hercules-ci-api/src/Hercules/API/Agents/AgentInfo.hs

```diff
@@ -0,0 +1,15 @@
+{-# LANGUAGE DeriveAnyClass #-}
+module Hercules.API.Agents.AgentInfo where
+
+import           Hercules.API.Prelude
+
+data AgentInfo = AgentInfo
```

Comment:
Hmm, all of this should be also sent on pings once they are implemented (as they will also cover agent restarts, that might change some of them)

Reply:
Yep, that's why I factored these attributes out.